adapter/pkg/messaging/azure_connection.go

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+// Package messaging holds the implementation for event listeners functions
+package messaging
+
+import (
+	logger "github.com/wso2/product-microgateway/adapter/internal/loggers"
+)
+
+var (
+	// AzureRevokedTokenChannel stores the revoked token events
+	AzureRevokedTokenChannel chan []byte
```

Comment:
Should we rename this to channel?

Reply:
+1.. We should use generic terms.

Reply:
We need channels for each topic like tokenRevocation, Notification etc and generic term would be for token revocation would be RevokedTokenChannel but this is already used in earlier implementation. Since azure_connection.go is in same package we cannot use that. wdyt ?

Reply:
How about asbRevokedTokenChannel? Because, using 'chan' here doesn't anyway resolve the problem. You'll face the same issue with the next topic.

Reply:
chan is a keyword and this how channels are getting created in go. eg- channel1 chan string, channel2 chan string  are two channels of string type.
Concern is the name AzureRevokedTokenChannel i guess